src/database/exiles_database.py

```diff
@@ -107,27 +134,29 @@ def get_user_exiles(user_id) -> List[tuple]:
         cursor.execute(query, params)
         res = cursor.fetchall()
 
-        return res
+        if res is not None:
+            return [Exile(*row) for row in res]  # Create a list of Exile objects
+        else:
+            return None
```

Comment:
Can this return an empty list instead of null

Reply:
Fixed in latest